frontend/src/Frontend/Transactions.hs

```diff
@@ -138,35 +190,91 @@ txTable
   :: (DomBuilder t m, Prerender js t m,
       RouteToUrl (R FrontendRoute) m, SetRoute t (R FrontendRoute) m)
   => NetId
+  -> Text
   -> [TxSummary]
   -> m ()
-txTable _ [] = blank
-txTable net txs = do
+txTable _ hdr [] = do
+  el "h4" $ text hdr
+  text "No results"
+txTable net hdr txs = do
+  el "h4" $ text hdr
   elClass "table" "ui compact celled table" $ do
     el "thead" $ el "tr" $ do
       el "th" $ text "Status"
       el "th" $ text "Chain"
+      el "th" $ text "Time"
       el "th" $ text "Height"
-      elClass "th" "two wide" $ text "Sender"
       el "th" $ text "Request Key"
+      el "th" $ text "Preview"
     el "tbody" $ do
       forM_ txs $ \tx -> el "tr" $ do
         let chain = _txSummary_chain tx
         let height = _txSummary_height tx
+        let rk = _txSummary_requestKey tx
         let status = case _txSummary_result tx of
                        TxSucceeded -> elAttr "i" ("class" =: "green check icon" <> "title" =: "Succeeded") blank
                        TxFailed -> elAttr "i" ("class" =: "red close icon" <> "title" =: "Failed") blank
                        TxUnexpected -> elAttr "i" ("class" =: "question icon" <> "title" =: "Unknown") blank
-        elAttr "td" ("class" =: "center aligned" <> "data-label" =: "Status") status
-        elAttr "td" ("data-label" =: "Chain") $ text $ tshow chain
-        elAttr "td" ("data-label" =: "Height") $ blockLink net (ChainId chain) height $ tshow height
-        elAttr "td" ("data-label" =: "Sender") $ senderWidget tx
-        elAttr "td" ("data-label" =: "Request Key") $ do
-          let contents = case (_txSummary_code tx, _txSummary_continuation tx) of
-                           (Just c, _) -> _txSummary_requestKey tx
-                           (_, Just v) -> showCont v
-                           (_, _) -> ""
-          text contents
+        elAttr "td" ("class" =: "center aligned" <> "data-label" =: "Status")
+            status
+        elAttr "td" ("data-label" =: "Chain") $
+            text $ tshow chain
+        elAttr "td" ("data-label" =: "Time") $
+            text $ T.take 19 $ tshow (_txSummary_creationTime tx)
+        elAttr "td" ("data-label" =: "Height") $
+            blockLink net (ChainId chain) height $ tshow height
+        elAttr "td" ("data-label" =: "Request Key") $
+            txDetailLink net rk (T.take 24 rk <> "...")
```

Comment:
I'd probably move the manual ellipsis into CSS with something like this:

```
  overflow: hidden;
  text-overflow: ellipsis;
```

...or similar.  But we can do that later.

Reply:
The advantage of doing it that way is that the user can still copy/paste the full value even if it doesn't fit.  See, for example https://github.com/kadena-io/chainweaver/blob/3ea0bb317c07ddf954d4ebf24b33d1be7d5f9c45/backend/sass/common.blocks/_wallet.scss#L47

Reply:
Yeah that's good stuff. We need it for bond explorer eliding too.